dist/debian/control.template

```diff
@@ -18,6 +18,8 @@ Conflicts: %{product}-server (<< 1.1)
 Package: %{product}-server
 Architecture: any
 Depends: ${misc:Depends}, %{product}-conf (= ${binary:Version}), %{product}-python3 (= ${binary:Version})
+Replaces: %{product}-tools (<<5.5)
+Breaks: %{product}-tools (<<5.5)
```

Comment:
This shouldn't be needed. There should be no files that are installed by both packages. It should be possible to install scylla-tools-java alongside scylla-server (although no longer necessary).

Reply:
Ah, it only breaks/replaces older versions. I think it's unnecessary, but it isn't harmful.

Reply:
There are no common files, of both packages are from the same version.
But scylla-core >= 5.5 will install 2 files (nodetool and nodetool-completion) that scylla-tools < 5.5 also installs. @tchaikov told me this is how I handle this in debian packaging (I myself have no idea what I'm doing here).

Reply:
@avikivity this is necessary for transitioning files from one package to another.

> There should be no files that are installed by both packages. 

this is not true.  as Botond put, they share the same two files.

Reply:
I don't know enough about debian to comment so I'll defer to @tchaikov. I extrapolated from my rpm knowledge.

Reply:
@denesb just wanted to double check with you: is the first version with this transition 6.0 or 5.5? IIUC, it should be 6.0, am i right?

Reply:
In master, the current version is still called 5.5~dev. Using 6.0 here, will cause the build to fail, see https://github.com/scylladb/scylladb/pull/18255?notification_referrer_id=NT_kwDOABUy2bMxMDI3NjcwMjE5MjoxMzg5Mjcz&notifications_query=reason%3Aparticipating#issuecomment-2064063333.

Reply:
i see. thanks.